src/global/variables/IntegerLiteral.java

```diff
@@ -0,0 +1,24 @@
+package global.variables;
+
+public class IntegerLiteral extends Clause {
```

Comment:
Do you think it makes sense to have a float/double literal as well? Not sure if there is a use case, but I'm wondering if we'd need it

Reply:
We were planning to add them already, I just wanted to get that ball rolling